rollback commit https://github.com/jamesmcm/vopono/commit/4d22055f478bb59f8254b4c6266db744a8abbab0

   --> vopono_core/src/config/providers/azirevpn/wireguard.rs:300:40
    |
300 |             let host_ip = host_lookup?.next().context(format!(
    |                                        ^^^^
    |

---

--- a/vopono_core/src/config/providers/azirevpn/wireguard.rs
+++ b/vopono_core/src/config/providers/azirevpn/wireguard.rs
@@ -296,11 +296,7 @@ impl WireguardProvider for AzireVPN {
                 log::error!("Could not resolve hostname: {}, skipping...", location.pool);
                 continue;
             }
-
-            let host_ip = host_lookup?.next().context(format!(
-                "DNS lookup for host '{}' returned no IP addresses",
-                location.pool
-            ))?;
+            let host_ip = host_lookup.unwrap().first().cloned().unwrap();
             log::debug!("Resolved hostname: {} to IP: {}", &location.pool, &host_ip);
             // TODO: avoid hacky regex for TOML -> wireguard config conversion
             let wireguard_peer = WireguardPeer {
